src/modules/email/adapters/__tests__/zoho-mail.adapter.spec.ts

```
@@ -188,8 +191,159 @@ describe('ZohoMailAdapter', () => {
+
+    beforeEach(() => {
+      jest.resetModules();
+      process.env = { ...originalEnv };
+    });
+
+    afterEach(() => {
+      process.env = originalEnv;
+    });
+
+    it('should use default logo URL when MAIL_LOGO_URL is not set', async () => {
+      delete process.env.MAIL_LOGO_URL;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)